src/main/msp/msp.c

```diff
@@ -2157,6 +2158,7 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         gpsRescueConfigMutable()->throttleHover = sbufReadU16(src);
         gpsRescueConfigMutable()->sanityChecks = sbufReadU8(src);
         gpsRescueConfigMutable()->minSats = sbufReadU8(src);
+        gpsRescueConfigMutable()->allowArmingWithoutFix = sbufReadU8(src);
```

Comment:
You will need to add a 
```
 if (sbufBytesRemaining(src) >= 1) {
```
And a comment indicating in which version it is being added. Check for similar examples in the same file that you are modifying, in the read and in the write part, in both.